deploy/helm/kubecf/values.yaml

```diff
@@ -225,6 +225,20 @@ services:
       start: 20000
       end: 20008
 
+router:
+  # tls is an array of objects containing the `crt` and `key` keys. E.g.
```

Comment:
Hmm, would the stuttering be slightly better if we flip them ("…containing the `key` and `crt` keys.")?

Reply:
```suggestion
  # tls is an array of objects containing the `key` and `crt` keys. E.g.
```